domain/src/main/kotlin/no/nav/su/se/bakover/domain/søknadsbehandling/Søknadsbehandling.kt

```diff
@@ -253,6 +265,15 @@ sealed class Søknadsbehandling : BehandlingMedOppgave, BehandlingMedAttestering
                     attesteringer,
                 ).right()
             }
+
+            override fun leggTilOppholdIUtlandet(
+                oppholdIUtlandet: OppholdIUtlandetVilkår,
+                clock: Clock,
+            ): Either<KunneIkkeLeggeTilOppholdIUtlandet, Vilkårsvurdert> {
+                return copy(
```

Comment:
Kunne vi hatt en funksjon som gjorde både copy og tilTilkårsvurdert? Lage en tilTilkårsvurdert med en ekstra param?

Reply:
Den vil uansett måtte ligge i hver enkelt subklasse for å få tak i riktig copy, men uansett en nyttig funksjon etterhvert som vi får flere ting over på dette formatet (da vi ønsker å vilkårsvurdere på nytt hver gang nye vilkår/grunnlag legges til).